tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -225,6 +225,39 @@ def objective(trial: optuna.Trial) -> float:
         assert target_config["dropout"] == dropout
 
 
+def test_dump_best_config_with_environment_variables() -> None:
+    with tempfile.TemporaryDirectory() as tmp_dir:
+
+        def objective(trial: optuna.Trial) -> float:
+            trial.suggest_float("DROPOUT", dropout, dropout)
+            trial.suggest_float("LEARNING_RATE", 1e-2, 1e-1)
+            executor = optuna.integration.AllenNLPExecutor(
+                trial,
+                input_config_file,
+                tmp_dir,
+                include_package="tests.integration_tests.allennlp_tests.tiny_single_id",
+            )
+            return executor.run()
+
+        dropout = 0.5
+        input_config_file = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            "example_with_environment_variables.jsonnet",
+        )
+        output_config_file = os.path.join(tmp_dir, "result.json")
+
+        os.environ["TRAIN_PATH"] = "tests/integration_tests/allennlp_tests/sentences.train"
+        os.environ["VALID_PATH"] = "tests/integration_tests/allennlp_tests/sentences.valid"
+
+        study = optuna.create_study(direction="maximize")
+        study.optimize(objective, n_trials=1)
+
+        optuna.integration.allennlp.dump_best_config(input_config_file, output_config_file, study)
+        best_config = json.loads(_jsonnet.evaluate_file(output_config_file))
+        assert os.getenv("TRAIN_PATH") == best_config["train_data_path"]
+        assert os.getenv("VALID_PATH") == best_config["validation_data_path"]
+
```

Comment:
Could you remove the environmental variables?
They can be referred by the other test cases and we may have unexpected side-effect.

```suggestion
        assert os.getenv("TRAIN_PATH") == best_config["train_data_path"]
        assert os.getenv("VALID_PATH") == best_config["validation_data_path"]

        del os.environ["TRAIN_PATH"]
        del os.environ["VALID_PATH"]
```

Reply:
I see. Thank you for the catch!